events/bingo/README.md

```diff
@@ -1,51 +1,105 @@
 # BINGO
-Ummm...well...it's bingo!
+A Bingo game for High Fidelity. Requires setting up a domain in a very specific way. There's no easy way to deploy this Bingo game on your own domain right now.
 
-## Release Notes
+## Secrets
 
-### v1.16
+The code for this game is open source. There are two sets of secrets that you must have set up for this game to work:
+1. A `config.json` file in `<bingo root>/config/config.json` that contains the data below.
+2. A `dbInfo.json` file in `<bingo root>/webApps/dbInfo.json` that contains the data below.
+
+### `config.json`
+Here's what your `config.json` file should look like:
+```
+{
+    "requestURL": "<a deployed version of the bingo.js Web App. MAKE SURE you also update REQUEST_URL in bingoCard_ui.js!>",
+    "usersAllowedToSpinWheel": [<an array of strings dictating who can spin the Bingo wheel>],
+    "gameAudioPosition": { "x": <int>, "y": <int>, "z": <int> },
+    "dbTablePrefix": "<a prefix for all of the Bingo tables in your DB associated with this deployment>",
+    "possiblePrizes": [<an array of short strings dictating the prizes that players can win>],
+    "debugMode": <true|false, depending on whether you want to enable various debug functions, such as the ability to use the Bingo Test App to call all numbers immediately>
+}
+```
+
+### `dbInfo.json`
+Here's what your `dbInfo.json` file should look like:
+```
+{
+    "mySQLHost": "<the host associated with your database>",
+    "mySQLUsername": "<MySQL DB username>",
+    "mySQLPassword": "<MySQL DB password>",
+    "databaseName": "<the name of the database associated with Bingo>"
+}
+```
+
+## Testing
+You can perform a local 1-person test of this new code by doing the following:
+1. Restore a Bingo content archive onto your local sandbox so that you have all of the necessary content.
+2. Set up your Secrets as per the  "Secrets" section above.
+3. Run the `bingo.js` Web app from a local NodeJS v10.15.1 installation
+    - Run `node bingo.js` from the `webApps` directory
+4. Run MySQL locally.
+    - You can create the necessary Bingo database by modifying the `startup()` function within the `bingo.js` web app script.
+5. Run a local HTTP server to serve the client/server scripts, then change all of the script URLs on the content.
+    - With NodeJS and `npm` installed, stand up a simple HTTP server with `npm install http-server -g` followed by `http-server -p 8888` from the `bingo` directory.
```

Comment:
You used to have python instructions here as well.  I thought that was nice and could be a good addition to an alternative to the node setup.

Reply:
the method by which a user stands up a local HTTP server for testing doesn't matter. this tip here is for convenience; it's not an instruction. i hear what you're saying - more alternatives are nice - but that kind of thing would belong in a bigger "how to test content locally" guide rather than the bingo readme.